terra_notebook_utils/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.0.0'
+__version__ = '0.1.0'
```

Comment:
is this actually used? When I pip install this, it tells me its on version `0.1.1`

Reply:
No, this is unused (good catch!). See https://github.com/DataBiosphere/terra-notebook-utils/pull/69#discussion_r440376144